### ISSUE TITLE
Add an SSL certificate

### ISSUE THREAD

**Issue by stephanmax** [Code-for-All/civicpatterns.org#35]
For some tender https love 😄 I am happy to help setting up letsencrypt—let me know!

**Comment by longhotsummer** [comment]
Good idea. We can easily use github page's SSL support. However, it says that civicpatterns.org is pointing at outdated IP addresses.

I'm not sure who has control over the domain to update it. @pudo ?

Once that is done, it's a matter of checking the SSL option in this repo's settings.

**Comment by stephanmax** [comment]
Sweet, that sounds straight-forward enough! Thanks, @longhotsummer!
